repo/packages/S/spark-notebook/0/marathon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "/spark-notebook",
+  "cpus": 0.5,
+  "mem": 4000,
```

Comment:
Does the notebook need 4GB of memory? What is happening on the notebook side vs the executor side?